### PR TITLE
Closes issue #21: Mac and Linux opening support

### PR DESCRIPTION
**Pull request by Compro-Prasad**
Using `open` for Mac and `xdg-open` for Linux/FreeBSD.

### CODE REVIEW

**Comment by pandusonu2** [comment]
Squash and resolve merge conflict :+1:

**Comment by Compro-Prasad** [comment]
Sorry, totally forgot that I had to update this one. Now its done.

File: BASH/downloader.sh [hunk 1]
```diff
@@ -102,10 +115,12 @@ do
...
+		nautilus $path
 	else
-	gnome-open chap$chapno.pdf	
- 	rm -rf $chap
-        fi
+		if [ -n "$openWith" ]; then
+			$openWith chap$chapno.pdf
+		fi
+		rm -rf $chap
+	fi
```
Comment (pandusonu2) [inline BASH/downloader.sh:125]: Indentation

Comment (Compro-Prasad) [reply BASH/downloader.sh:125]: I don't see a problem. Can you elaborate a bit more mentioning the line number.

Comment (pandusonu2) [reply BASH/downloader.sh:125]: It just looked wrong in the git diff view.

My bad, I was checking diff lines, it looks fine, you can merge it :) would like to clear all PRs for the moment before refactoring the entire repository

**Pull request merged**